include/ccf/ds/logger.h

```diff
@@ -271,6 +271,7 @@ namespace logger
 
     static inline void default_init()
     {
+      get_loggers().clear();
```

Comment:
Unrelated change: I noticed that we would get duplicate logs if we called `logger::config::default_init()` multiple times, which doesn't seem intended behaviour?

Reply:
That definitely sounds like a bug.